frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -127,6 +129,16 @@ const RealPhoneForm = (props: RealPhoneFormProps) => {
     </div>
   );
 
+  const onChange: ChangeEventHandler<HTMLInputElement> = (e) => {
+    // remove all non-digit characters
+    const phone = e.target.value.replace(/\D/g, "");
+
+    // set the new value if under 10 digits
+    if (phone.length <= 10) {
```

Comment:
Question: why wouldn't we want to update the state if the input length is less than 10 digits?

Reply:
Setting max/min length to 16 to include formatting characters "(234) 342 - 2342".length === 16